app/views/record/_access_buttons.html.erb

```diff
@@ -13,4 +13,10 @@
       View <%= @record['provider'] %> record
     </a>
   <% end %>
+  <% if access_type(@record) != 'unknown: check with owning institution' &&
```

Comment:
This is fine as it was like this before moving the location of where it shows up, but I'm curious if we actually need the check to make sure it is an MIT record or if we should just always display the download metadata link if it is present. Not blocking, but suspect we're being overly cautious here.

Reply:
Yeah, good point. I think this made more sense when that value was 'Not owned by MIT'. Now that the value itself doesn't make sense, it's probably best just to use `provider`.

Reply:
I don't think we should check either. I think we should just display this link if it exists.

Reply:
Oh. You're probably right about that, too.

I'd like to revisit that when we (hopefully) talk about displaying geodata file download links for all records, post-MVP. For now I agree it's fine as-is.